src/main/java/com/dezorganizacja/bridge/presenter/MainPresenter.java

```diff
@@ -2,8 +2,11 @@
 
 import com.dezorganizacja.bridge.domain.Message;
 
-public class MainPresenter {
+import java.util.Observable;
+
+public class MainPresenter extends Observable {
     private final Message message;
+    private String state;
```

Comment:
1. nie musi to być bijekcja (licytacja i sama gra może korzystać z tej samej sceny tylko zmieniać szczegóły, jak co jest klikalne itp.)
2. enumy, jak najbardziej, mogą być zamiast Stringów